index.js

```diff
@@ -598,6 +598,8 @@ alexa.app = function(name) {
             if (typeof self.sessionEndedFunc == "function") {
               return Promise.resolve(self.sessionEndedFunc(request, response));
             }
+          } else if (requestType && requestType.indexOf('Connections') > -1) {
```

Comment:
I am going to nitpick a bit. It's unclear what this does, so I would like the method to move into `request` and be similar to `isAudioPlayer` explaining what kind of request this is. Then, it should be checking that it starts with `Connections.` otherwise some future `WhateverConnections` will cause undebuggable behavior. 

Thoughts?

Reply:
can do that. can you point me to the line you're referring to with `isAudioPlayer`?